crates/solvers-dto/src/solution.rs

```diff
@@ -57,6 +57,8 @@ pub struct JitTrade {
     pub order: JitOrder,
     #[serde_as(as = "HexOrDecimalU256")]
     pub executed_amount: U256,
+    #[serde_as(as = "Option<HexOrDecimalU256>")]
+    pub fee: Option<U256>,
```

Comment:
If we go with this option, I think we should remove `order.fee_amount` (as this field will have to always be 0 or will be ignored) and make this field here non optional (solvers can specify a 0 fee)